src/components/SignUp/Credits.tsx

```diff
@@ -16,6 +69,93 @@ const Credits: React.FC<{
           <Tagline>Where might we have seen you?</Tagline>
         </Col>
       </Row>
+
+      <Row>
+        <Col lg="4">
+          <Form>
+            <InputField
```

Comment:
All of your input fields need to actually go into your pastPerformances.map loop